aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConfiguration.java

```diff
@@ -71,6 +74,8 @@ private DataStoreConfiguration(Builder builder) {
         this.syncIntervalInMinutes = builder.syncIntervalInMinutes;
         this.syncExpressions = builder.syncExpressions;
         this.doSyncRetry = builder.doSyncRetry;
+        this.syncMaxConcurrentModels = builder.syncMaxConcurrentModels != null ?
```

Comment:
This has to be nullable in the builder because of the weird user provided thing, right?

Reply:
Yep